domains/Accounts/Controllers/AccountsStoreController.php

```diff
@@ -11,13 +11,6 @@
 
 class AccountsStoreController extends Controller
 {
-    private User $user;
-
-    public function __construct(User $users)
-    {
-        $this->users = $users;
-    }
-
```

Comment:
I think I understand why this was removed. But I think we should compose our controllers and inject any dependency and reference them when needed. 

But we should do it on another PR, to not pollute this one with code that has little to do with it. Can you revert this change, for now?

Reply:
sure